### ISSUE TITLE
V2 webpage not displaying properly

### ISSUE THREAD

**Issue by eeicc** [opencve/opencve#472]
What values should be added?
I have changed the setting of ALLOWED_HOSTS to local or cleared IP.
It's still the same.

![螢幕擷取畫面 2024-12-04 112233](https://github.com/user-attachments/assets/020bc14d-b16c-4f36-9702-653b05150a7a)

**Comment by ncrocfer** [comment]
First of all **Hello** 😉 

Without more information we can't help you. For instance how did you install OpenCVE? Did you use the `install.sh` script?

**Comment by eeicc** [comment]
Yes, I followed the deployment file, ran the commands in order, and finally typed in “. /install.sh import-opencve-kb” and the content finally appeared, but not as expected.

The problem is the same as V1, but V1 can display the content correctly by clearing the IP.

**Comment by ncrocfer** [comment]
Closing this issue as without any logs it's complicated to help.

Try to simply launch `install.sh` script, it will work as it.
